docs/changelog.rst

```diff
@@ -164,9 +169,9 @@ Thanks to Michał Górny in `PR #315 <https://github.com/adamchainz/time-machine
 2.3.0 (2021-07-05)
 ------------------
 
-* Allow passing ``tick`` to ``Coordinates.move_to()`` and the pytest fixture’s
-  ``time_machine.move_to()``. This allows freezing or unfreezing of time when
-  travelling.
+* Allow passing ``tick`` to ``Traveller.move_to()`` (or ``Coordinates.move_to()`` until version 2.17.0)
+  and the pytest fixture’s ``time_machine.move_to()``. This allows freezing or unfreezing
+  of time when travelling.
```

Comment:
[opinion] I would leave this as is. As it's historical

Reply:
yes please don't edit old chaneglog notes